web/migrations/0054_remove_featurevote_unique_ip_feature_vote_and_more.py

```
@@ -0,0 +1,77 @@
+            field=web.crypto_utils.EncryptedTextField(),
+        ),
+        migrations.AlterField(
+            model_name="profile",
+            name="bio",
+            field=web.crypto_utils.EncryptedTextField(blank=True, max_length=500),
+        ),
+        migrations.AlterField(
+            model_name="profile",
+            name="discord_username",
+            field=web.crypto_utils.EncryptedCharField(
+                blank=True, help_text="Your Discord username (e.g., User#1234)", max_length=50
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)